coderd/workspaceagents.go

```diff
@@ -370,12 +376,12 @@ func (api *API) workspaceAgentPTY(rw http.ResponseWriter, r *http.Request) {
 		})
 		return
 	}
-	defer func() {
-		_ = conn.Close(websocket.StatusNormalClosure, "ended")
```

Comment:
Is this `"ended"` message important?

Calling `wsNetConn.Close()` bellow will simply close it with `(StatusNormalClosure, "")`.